frontend/docs/angular.mdx

```diff
@@ -0,0 +1,107 @@
+---
+id: angular
+title: Angular
+hide_title: false
+hide_table_of_contents: false
+keywords:
+  - final space
+  - finalspace
+  - finalspaceapi
+  - final space wiki
+  - api
+  - restapi
+  - rest api
+  - example
+  - angular
+description: A live example of the Final Space API using Angular.
+---
+
+
+<Tabs
```

Comment:
We don't need this tabs.

Since there is only tab
Can you remove this ?

Reply:
ok, i remove it